### DISCUSSION TITLE
Error Saat di Hosting

### DISCUSSION THREAD

**Discussion by Bisbissdev** [kirimwa-id/forum#64] [Tanya Jawab] [answered]
Halo semua, mohon bantuan, ketika saya integrasikan api di local lancar lancar saja, tapi ketika pas di hosting kenapa error ya, Terimakasih
![error](https://user-images.githubusercontent.com/93394155/140475327-87a0895c-82a8-4418-9e39-53418b4ddca3.png)

**Comment by rioastamal** [comment, marked as answer]
- API apa yang di hit?
- Response error dari API KirimWA.id seperti apa? 

Agar jawaban dapat segera didapat baiknya mengikut format dari https://github.com/kirimwa-id/forum/discussions/2

**Reply by Bisbissdev** [reply]
Sudah berhasil mas, ternyata setting cpanel nya, kemarin saya hubungin cs nya untuk setting php.ini di cpanel nya, sekarang sudah berjalan lancar…
Terimakasih
